internal/version/version.go

```diff
@@ -0,0 +1,8 @@
+package version
+
+// GitCommit indicates which commit the rukpak binaries were built from
+var GitCommit string
+
+func String() string {
+	return GitCommit
+}
```

Comment:
I don't see this used anywhere. Remove it? Seems like anything that needs to know the git commit can just use `version.GitCommit` directly.